src/pages/demo.md

```diff
@@ -10,5 +10,5 @@ keywords:
   - C++
   - Scripting
 title: Photoshop API Demo
-frameSrc: https://easybake.adobedice.com?ref=home
+frameSrc: https://easybake-stage.adobedice.com?ref=home
 ---
```

Comment:
How we take care of environments? I mean when we deploy to prod?

Reply:
There is no environment variable set up for this repo, so we need to deploy separately.  This branch is for stage only.